src/main/java/com/entropyteam/entropay/employees/models/Contract.java

```diff
@@ -0,0 +1,124 @@
+package com.entropyteam.entropay.employees.models;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import com.entropyteam.entropay.common.BaseEntity;
+import com.entropyteam.entropay.employees.dtos.ContractDto;
+
+@Entity(name = "Contract")
+@Table(name = "contract")
+public class Contract extends BaseEntity {
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "company_id")
+    private Company company;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "employee_id")
+    private Employee employee;
+
+    @Column
+    private LocalDate startDate;
+    @Column
+    private LocalDate endDate;
+    @Column
+    private String role;
+    @Column
+    private String seniority;
```

Comment:
I think that it will be better to have Role and Seniority as enums

Reply:
Position(role) and seniority added as entities.